parsers/nmparser/set_missing_values_to_default_test.go

```diff
@@ -128,22 +116,21 @@ func TestSetMissingValuesToDefaultParameterDataValues(tt *testing.T) {
 					},
 				}},
 			},
-			etaCount: 3,
 		},
 	}
 
 	for _, test := range tests {
 		tt.Run(test.name, func(tt *testing.T) {
 			t := wrapt.WrapT(tt)
 
-			setMissingValuesToDefault(&test.modelOutput, test.etaCount, test.epsCount)
-			t.R.Equal(7, len(test.modelOutput.ParametersData[0].StdErr.Theta), "Fail :"+test.context)
-			t.R.Equal(2, len(test.modelOutput.ParametersData[0].RandomEffectSD.Sigma), "Fail :"+test.context)
-			t.R.Equal(3, len(test.modelOutput.ParametersData[0].RandomEffectSDSE.Omega), "Fail :"+test.context)
+			setMissingValuesToDefault(&test.modelOutput)
+			t.R.Equal(7, len(test.modelOutput.ParametersData[0].StdErr.Theta), "Fail :"+test.name)
+			t.R.Equal(2, len(test.modelOutput.ParametersData[0].RandomEffectSD.Sigma), "Fail :"+test.name)
```

Comment:
test.name is covered in all of these cases by the tt.Run above. it is reported as `Test<Name>/7_thetas_3_omegas_2_sigmas`

Reply:
ah, so can we just get rid of that `, "Fail :"+test.name` part of each line?

Reply:
yup!